src/views/Correction.vue

```diff
@@ -327,9 +327,8 @@ export default class Correction extends Mixins(CommonMixin, DateMixin, EnumMixin
 
   /** AGM Year of original filing (AR only). */
   get agmYear (): string {
-    if (this.origFiling?.annualReport?.annualReportDate) {
-      const date: string = this.origFiling.annualReport.annualReportDate
-      return date.slice(0, 4)
+    if (this.origFiling?.annualReport) {
+      return this.origFiling.header.ARFilingYear.toString()
```

Comment:
`annualReportDate` is the date the original filing was filed.

What we want is `ARFilingYear`.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/c983e80d-199a-4237-9f4b-1f6e2c561935)